package.json

```diff
@@ -43,84 +43,93 @@
     "url": "https://github.com/opensearch-project/oui.git"
   },
   "resolutions": {
-    "**/trim": "0.0.3",
-    "**/axios": "^0.21.1",
-    "**/ansi-html": "^0.0.8",
```

Comment:
I dont see this resolution anymore

Reply:
> I dont see this resolution anymore

"\*\*/trim" -> "remark-parse/trim": "0.0.3"
"\*\*/axios" -> "codesandbox/axios": "^0.21.1"
"\*\*/ansi-html" is not used